.travis.yml

```diff
@@ -14,15 +14,12 @@ sudo: false
 # to alert us to any real upcoming change in the language that would be
 # incompatible with our existing code.
 #
-# We may choose to test additional versions as time allows,
-# as long as the number of entries in the build matrix doesn't exceed ten.
-# These additional versions will not be required, so they're advisory only.
-# Since Travis runs five jobs at a time, a limit of ten jobs makes sense.
+# We used to test additional versions beyond the last two,
+# but since Travis only runs five jobs at a time throughout all of Exercism,
+# it is better citizenship to not test the additional versions.
 go:
-  - 1.7
-  - 1.6.3
-  - 1.5.4
-  - 1.4.3
+  - 1.8
+  - 1.7.5
```

Comment:
Ah, okay. I agree (weird to have 1.7.5 between 1.8 and tip) and now's a good time to change.